### ISSUE TITLE
False positive on sqlalchemy import error

### ISSUE THREAD

**Issue by rmariano** [python/mypy#2952]
I have something like

```python
from sqlalchemy.orm import scoped_session
```

And I get the error:

```
 "module" has no attribute "scoped_session"
```

Buit it *does* have this module, so it should not raise this error.
mypy version is 0.501 (latest at the time of this writing)

**Comment by JelleZijlstra** [comment]
Thanks for reporting this! This is a typeshed issue; the name is missing from https://github.com/python/typeshed/blob/master/third_party/2and3/sqlalchemy/orm/__init__.pyi. Could you open an issue (or even submit a PR) against typeshed?

**Comment by ethanhs** [comment]
Discussion of this issue is continuing in the typeshed tracker here: https://github.com/python/typeshed/issues/974

**Comment by gvanrossum** [comment]
Since it's really a typeshed issue, closing.
